shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/presentation/helpers/MokoLocalize.kt

```diff
@@ -0,0 +1,17 @@
+package co.nimblehq.blisskmmic.presentation.helpers
+
+import co.nimblehq.blisskmmic.MR
+import dev.icerock.moko.resources.AssetResource
+import dev.icerock.moko.resources.ResourceContainer
+import dev.icerock.moko.resources.StringResource
+import dev.icerock.moko.resources.desc.Resource
+import dev.icerock.moko.resources.desc.StringDesc
+
+class MokoLocalize {
+    fun localize(id: String?): StringDesc? {
+        if (id == MR.strings.common_error.toString()) {
+            return StringDesc.Resource(MR.strings.common_error)
+        }
+        return null
+    }
```

Comment:
Should we name this function to the below suggestion as it only supports `common_error`?
```suggestion
    fun localizedError(id: String?): StringDesc? {
        if (id == MR.strings.common_error.toString()) {
            return StringDesc.Resource(MR.strings.common_error)
        }
        return null
    }
```

Reply:
This extension is allow to localized any needed id, from the id's string description. I think it's best to name it generically. Currently there's only `common_error` that needed this feature but any string that is dynamically localized from KMM module needs to have this feature.

Reply:
ah okay, it makes sense now 👌